### DISCUSSION TITLE
how lunarvim automatically downloads LSPs in mason

### DISCUSSION THREAD

**Discussion by shubhisroking** [LunarVim/LunarVim#4510] [General]
While setting up my own config I was missing that automatic LSP downloading of lunar vim through mason, I digged into docs but didn't find anything related to automatic installation of LSPs through mason

**Comment by olrtg** [comment]
I use https://github.com/WhoIsSethDaniel/mason-tool-installer.nvim for this purpose.

**Reply by shubhisroking** [reply]
thanks a lot
